diesel_cli/src/main.rs

```diff
@@ -191,22 +221,23 @@ fn redo_latest_migration<Conn>(conn: &Conn) where
     }).unwrap_or_else(handle_error);
 }
 
-fn handle_error<E: Error>(error: E) {
+fn handle_error<E: Error, T>(error: E) -> T {
```

Comment:
Oh neat, I didn't know the compiler would let this slide. Does this emit an `unreachable_code` warning?